ocs_ci/ocs/node.py

```diff
@@ -301,6 +302,45 @@ def add_new_node_and_label_it(machineset_name):
     return new_spun_node[0]
 
 
+def add_new_node_and_label_upi(config, num_nodes):
+    initial_nodes = tests.helpers.get_worker_nodes()
+    plt = PlatformNodesFactory()
+    node_util = plt.get_nodes_platform()
+    if config.ENV_DATA.get('rhel_workers'):
+        logging.info('AWS UPI RHEL adding node')
+        node_util.create_and_attach_nodes_to_cluster(
+            {}, 'RHEL', num_nodes)
+    else:
+        logging.info('AWS UPI RHCOS adding node')
+        node_util.create_and_attach_nodes_to_cluster(
+            {}, 'RHCOS', num_nodes)
+    for sample in TimeoutSampler(
+        timeout=600, sleep=6, func=tests.helpers.get_worker_nodes
+    ):
+        if len(sample) == len(initial_nodes) + num_nodes:
+            break
+
+    nodes_after_exp = tests.helpers.get_worker_nodes()
+    wait_for_nodes_status(
+        node_names=tests.helpers.get_worker_nodes(),
+        status=constants.NODE_READY
+    )
+
+    new_spun_nodes = list(set(nodes_after_exp) - set(initial_nodes)
+                          )
+    node_obj = ocp.OCP(kind='node')
+
+    for new_spun_node in new_spun_nodes:
+        node_obj.add_label(
+            resource_name=new_spun_node,
+            label=constants.OPERATOR_NODE_LABEL
+        )
+        logging.info(
+            f"Successfully labeled {new_spun_node} with OCS storage label"
+        )
+    return True
+
+
```

Comment:
make labeling node as optional

Reply:
Hey @prsurve , why labeling node as optional?

Reply:
Its because not every time we need for ocs we can make use of newly added node to run workloads or any specific pods, what you can do is mark_for_ocs_label=True and pass as param to function if someone doesn't need they can pass False so the function will not label it for ocs

Reply:
ack